### ISSUE TITLE
Automatic installation of updates for non-App Store builds of the Mac app

### ISSUE THREAD

**Issue by gregorymartin** [home-assistant/iOS#1243]
This is more a feature request than a bug, so apologies if this is the wrong forum for it.  I'm wondering if it would be possible to leverage a framework like [Sparkle](https://sparkle-project.org) to handle easier app updates when there are changes.  The current method of manual download from Github is not ideal.  Thanks and loving the app!

**Comment by TomBrien** [comment]
I believe the plan is to eventually move the app on to the App Store. I'm sure Zac will comment when he has a chance

**Comment by zacwest** [comment]
I've looked into it a bit; it's tough to get Sparkle working in a Catalyst app. I added a route we can use to call directly into AppKit methods, but for it to be able to update a sandboxed app also requires some in-beta Sparkle 2.x changes that are both complicated and convoluted.

Ideally we keep the betas running via GitHub releases, possibly using Sparkle to auto-update, and also post the app on the Mac App Store for the stable build.
